MAUI/Carousel-View/SwipeEvents.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: Swipe Events in .NET MAUI Carousel View control | Syncfusion
+description: Learn here all about Swipe Events support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+platform: maui
+control: Carousel
+documentation: ug
+---
+
+# Swipe Events in .NET MAUI Carousel View (SfCarousel)
+
+## Swipe Start
+
+The `SwipeStarted` event will be triggered when swipe gesture is started.
+
+`SwipeStartedEventArgs` contains `IsSwipeLeft` value, which denotes whether the swipe direction is from left side or not.
+
+## Swipe End
+
+The `SwipeEnded` event can be triggered after the swiping completed in the carousel items.
```

Comment:
The `SwipeEnded` event can be triggered after the swiping is completed in the carousel items.

Reply:
Addressed the changes.